pkgs/applications/editors/emacs/elisp-packages/manual-packages/calfw/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  trivialBuild,
+  howm,
+}:
+
+trivialBuild {
+  pname = "calfw";
+  version = "0-unstable-2025-05-02";
+  src = fetchFromGitHub {
+    owner = "haji-ali";
+    repo = "emacs-calfw";
```

Comment:
I am not sure about **replacing** `calfw` with a fork, especially since that fork seems not popular.

In addition, we try to keep the number of `manual-packages` small.

So I think we should first try to get this fork on MELPA by communicating with the author of this fork, maintainers of MELPA and maybe the author of the original `calfw`.  This fork can use the name `calfw` on MELPA, which means it will replace the original `calfw`, or it can use some other name, which means it will not replace the original `calfw`.  Note that MELPA packages will be added to Nixpkgs automatically.

If that will not happen for some reason, I think we can add this fork to `manual-pakcages` in Nixpkgs using a name other than `calfw`, i.e., not replacing the original `calfw`.

Reply:
I think the fork might just lack discoverability since the maintainer (to my knowledge) never officially announced to take up maintainance of the package. However, I have been following the fork for some years and they are still regularly pushing patches, so it seems they are doing just that.

I will message the author of the fork whether they would want it added to MELPA; if yes, I will then continue to ping the original author and MELPA.

Reply:
Great!